omod/src/main/webapp/resources/scripts/queue.js

```diff
@@ -48,7 +48,7 @@ var updateSearchResults = function(results){
 	_.each(searchResultsData, function(result){
 		var patient_name = result.patientName.replace("null","");
 		console.log("The results 1 are >>"+result);
-		if (result.referralConcept && result.referralConcept.conceptId === 2548){
+		if (result.referralConcept && result.referralConcept.conceptId === 165450){
```

Comment:
Also this, do we need to create those as new concepts?

Reply:
@ningosi it was already existing ID: 165445

Reply:
this is a new concept already shared with @Bushisky to create.

Reply:
Ok